plugins/labhub.py

```diff
@@ -222,6 +222,18 @@ def create_issue_cmd(self, msg, match):
                 target=user,
             )
 
+    @staticmethod
+    def profile_link(msg):
+        gitter_username = msg.frm.nick
+        pattern = '^(.*)(_.*?)$'
```

Comment:
you should "declare" where it's used

Reply:
I don't quite seem to understand what do you want me to declare :)

Reply:
You assign to pattern although it's just use in one of the if branches. You should do the assignment there^^

Reply:
Done :+1: